util/src/main/i18n/templates/map.properties

```diff
@@ -134,3 +134,13 @@ vote.limit = Selection limit for the vote has been reached
 
 vote.title.map = Selected Maps
 
+# {0} = player name (who toggled mode)
+# {1} = mode name
+vote.toggle = {0} toggled vote mode to {1}
+
+vote.mode.replace = Replace
+
+vote.mode.override = Override
```

Comment:
Replace and override basically mean the same thing, can we use different terms that make it more clear?

Reply:
@AustinLMayes suggested `create` as an alternative for override 🎉 
If you have different terms you wanted just let me know and I’ll be glad to adjust it.

Otherwise should be good to go now 👍